src/incrementer.js

```diff
@@ -201,23 +198,37 @@ var Incrementer = {
      * @param {jQueryElement} $elt - plugin's element
      * @fires Incrementer#plus.incrementer
      */
-    _inc: function($elt) {
+    _inc: function _inc($elt) {
         var options = $elt.data(dataNs),
             current = parseFloat($elt.val() || 0),
             value;
-
         value = gamp.add(current, options.step);
+
+        if ($elt[0].getAttribute('name') === 'normalMaximum') {
+            options.max = parseFloat($elt.parent().parent().find("[name='normalMaximum']").attr('value')) + 1;
```

Comment:
Would it be better to store $elt.parent().parent().find("XX").attr('value') inside a variable in order to try and tidy things up a bit?

Reply:
Or a getter function if it can be dynamically changed since we don't have reactivity here

Reply:
Done with @404passionFruit comment